src/main/java/de/tum/in/www1/artemis/domain/GradeStep.java

```diff
@@ -27,6 +27,8 @@ public class GradeStep extends DomainObject {
 
     private static final Logger log = LoggerFactory.getLogger(GradeStep.class);
 
+    public static final String PLAGIARISM_GRADE = "U";  // "U" stands for "Unterschleif"
```

Comment:
Maybe for a followup: I'm not sure if this is used generally or if different universities have different identifiers here. If it's the second case, it should be configurable in the course / exam configuration.

Reply:
@krusche mentioned that we can have "U" as plagiarism grade. I am not sure if this is a standard but the main intent is indicating that there is an extraordinary situation so in that sense I think this should work well for now. But I agree that making it configurable can be a nice enhancement that can be handled in a followup as you mentioned.